src/plugins/console/public/application/containers/editor/editor.tsx

```diff
@@ -11,7 +11,8 @@ import { debounce } from 'lodash';
 import { EuiProgress } from '@elastic/eui';
 
 import { EditorContentSpinner } from '../../components';
-import { Panel, PanelsContainer } from '../../../../../kibana_react/public';
+import { Panel } from '../../containers/split_panel/panel';
```

Comment:
Can you, please, add the `index.ts` file with exports of components from `containers/split_panel` and add the exports of `split_panel` components from `containers/index.ts`? 
And after, could you change the imports to the `containers`? It seems to me, the code will become much cleaner.